include/barebox.mk

```diff
@@ -0,0 +1,102 @@
+PKG_NAME ?= barebox
+
+ifndef PKG_SOURCE_PROTO
+PKG_SOURCE = $(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL = \
+	http://sources.lede-project.org \
```

Comment:
This line is unnecessary, it's handled by buildbot.

Reply:
Yes,I just refer to u-boot.mk.